tests/std/tests/P0896R4_ranges_iterator_machinery/test.cpp

```diff
@@ -352,50 +357,48 @@ struct weakly_incrementable_archetype : destructible_archetype<I>,
                                         weakly_incrementable_archetype_dt<I>,
                                         increment_ops<I, weakly_incrementable_archetype<I>, void> {
     // clang-format off
-    weakly_incrementable_archetype() requires (I != 1) {}
     weakly_incrementable_archetype(weakly_incrementable_archetype const&)                       = delete;
-    weakly_incrementable_archetype(weakly_incrementable_archetype&&) requires (I < 2 || I >= 4) = default;
+    weakly_incrementable_archetype(weakly_incrementable_archetype&&) requires (I < 1 || I >= 3) = default;
 
     weakly_incrementable_archetype& operator=(weakly_incrementable_archetype const&) = delete;
-    weakly_incrementable_archetype& operator=(weakly_incrementable_archetype&&) requires (I < 4 || I >= 6) = default;
+    weakly_incrementable_archetype& operator=(weakly_incrementable_archetype&&) requires (I < 3 || I >= 5) = default;
     // clang-format on
 };
 
-inline constexpr std::size_t weakly_incrementable_archetype_max = 12;
+inline constexpr std::size_t weakly_incrementable_archetype_max = 11;
 
 template <std::size_t I>
 struct incrementable_archetype : weakly_incrementable_archetype<I>,
                                  increment_ops<I, incrementable_archetype<I>, incrementable_archetype<I>> {
-    SEMIREGULAR_OPS(incrementable);
+    SEMIREGULAR_OPS(incrementable, 11);
     using increment_ops<I, incrementable_archetype<I>, incrementable_archetype<I>>::operator++;
 
     // clang-format off
-    bool operator==(incrementable_archetype const&) const requires (I != weakly_incrementable_archetype_max);
+    bool operator==(incrementable_archetype const&) const requires (I != 12);
     bool operator!=(incrementable_archetype const&) const
-        requires (I == weakly_incrementable_archetype_max + 1) = delete;
+        requires (I == 13) = delete;
     // clang-format on
 };
 
 inline constexpr std::size_t incrementable_archetype_max = 14;
 
 template <std::size_t I>
 struct iterator_archetype : weakly_incrementable_archetype<I> {
-    SEMIREGULAR_OPS(iterator);
+    COPYABLE_OPS(iterator);
 
-    // clang-format off
-    iterator_archetype& operator++() requires (I > 10);
-    void operator++(int) requires (I != 11);
+    iterator_archetype& operator++() requires(I > 9);
+    void operator++(int) requires(I != 10);
 
-    void operator*() requires (I == 12);
-    int operator*() requires (I != 12);
+    void operator*() requires(I == 11);
+    int operator*() requires(I != 11);
     // clang-format on

```

Comment:
The `// clang-format off` above (would have been right line 389) is being removed, but this `// clang-format on` is being retained. It appears that the space is being lost after `requires`, contrary to your preference. Should the `// clang-format off` and the spaces be restored?

Reply:
Yes, I find it convenient that `requires [^{]` introduces a _requires-clause_ and `requires(` or `requires {` introduce _requires-expressions_. Fixing. (Aha, and there's a regression in `P0784R7_library_support_for_more_constexpr_containers` as well.)